src/Data/DataSource.php

```diff
@@ -4,10 +4,12 @@
 use GraphQL\Type\Definition\ResolveInfo;
 use GraphQLRelay\Relay;
 
+use PHPUnit\Runner\Exception;
```

Comment:
@hughdevore I don't think this is supposed to be here, eh?

Reply:
Hmmm, dunno how that got added, will remove!